components/TextField/src/index.scss

```diff
@@ -0,0 +1,86 @@
+@import "~@utrecht/components/textbox/bem";
+
+.denhaag-textfield {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+}
+
+.denhaag-textfield--focus-within::after,
+.denhaag-textfield:focus-within::after {
+  border: var(--denhaag-focus-border);
+  border-radius: var(--denhaag-border-radius);
+  content: "";
+  display: block;
+  position: absolute;
+  top: -2px;
+  right: -2px;
+  bottom: -2px;
+  left: -2px;
```

Comment:
@bddjong 

Just thought of something (bit late, i know), but this only really works in our specific scenario. To fix it with a different set of design tokens this should be: 

`calc(-1 * var(--denhaag-focus-border-width))` instead of a plain old `-2px`.